MapboxNavigationTests/InstructionsBannerViewIntegrationTests.swift

```diff
@@ -93,33 +93,28 @@ class InstructionsBannerViewIntegrationTests: XCTestCase {
         resetImageCache()
     }
 
-    //FIXME: this test is artificially slow as we are polling the run loop following each simulated download. need a better way of synchronizing on the underlying completions.
     func testDelimiterDisappearsOnlyWhenAllShieldsHaveLoaded() {
         let view = instructionsView()
         
-        //setup expectations
         let firstExpectation = XCTestExpectation(description: "First Component Callback")
         let secondExpectation = XCTestExpectation(description: "Second Component Callback")
-        
-        view.primaryLabel.imageRepository = imageRepository
```

Comment:
Hey, what's going on here? Without these two lines, `InstructionLabel` will be hitting the actual `.shared` imageRepository. Am I missing something?

Reply:
Yeah, this test is still using the shared instance configured to use the `DownloadOperationSpy` class. It seems to be working well with the label's `.imageDownloadCompletion` pattern as a synchronization point. Let's see how it does. We can continue to refactor as needed since this is just a checkpoint.

Reply:
It's also worth noting that this assignment was assigning the same instance back onto the labels.